--- origsrc/heap-buddy-0.2/profiler/Makefile.am	2006-09-05 16:07:24.000000000 -0500
+++ src/heap-buddy-0.2/profiler/Makefile.am	2007-02-06 23:09:03.531250000 -0600
@@ -10,6 +10,7 @@
 	outfile-writer.c			\
 	heap-buddy.c
 
+libmono_profiler_heap_buddy_la_LDFLAGS = -avoid-version -no-undefined
 libmono_profiler_heap_buddy_la_LIBADD = @PROFILER_LIBS@
 
-INCLUDES = @PROFILER_CFLAGS@ -Wall
\ No newline at end of file
+INCLUDES = @PROFILER_CFLAGS@ -Wall
