diff -urN -x CYGWIN-PATCHES -x 'aclocal.m4*' -x ltmain.sh -x 'config.*' -x depcomp -x install-sh -x missing -x mkinstalldirs -x autom4te.cache -x '*compile' -x Makefile.in.in -x 'intltool*.in' -x 'xml-i18n-*.in' -x '*.pyc' -x '*.mo' -x '*.gmo' -x ABOUT-NLS -x Makevars.template -x COPYING -x INSTALL -x mdate-sh -x '*.orig' -x '*.rej' -x '*~' -x '*.temp' -x texinfo.tex -x ylwrap -x Makefile.in -x makefile.in -x configure -x gnome-doc-utils.make -x gnome-doc-utils.m4 -x intltool.m4 -x omf.make -x xmldocs.make origsrc/heap-buddy-0.2/profiler/Makefile.am src/heap-buddy-0.2/profiler/Makefile.am
--- origsrc/heap-buddy-0.2/profiler/Makefile.am	2006-09-05 16:07:24.000000000 -0500
+++ src/heap-buddy-0.2/profiler/Makefile.am	2007-02-06 23:09:03.531250000 -0600
@@ -10,6 +10,7 @@
 	outfile-writer.c			\
 	heap-buddy.c
 
+libmono_profiler_heap_buddy_la_LDFLAGS = -avoid-version -no-undefined
 libmono_profiler_heap_buddy_la_LIBADD = @PROFILER_LIBS@
 
-INCLUDES = @PROFILER_CFLAGS@ -Wall
\ No newline at end of file
+INCLUDES = @PROFILER_CFLAGS@ -Wall
